packages/core-data/src/hooks/test/use-query-select.js

```diff
@@ -17,9 +17,16 @@ import { render, screen, waitFor } from '@testing-library/react';
  */
 import useQuerySelect from '../use-query-select';
 
+// eslint-disable-next-line eslint-comments/disable-enable-pair
```

Comment:
Instead of disabling this rule, doesn't it make sense to just enable it at the end of the file? 

Basically, adding this to the EOF:

```
/* eslint-enable @wordpress/wp-global-usage */
```